readthedocs/templates/projects/project_advertising.html

```diff
@@ -72,7 +72,7 @@ <h4>{% trans "Opting out" %}</h4>
       {% endblocktrans %}
 
       <form method="get" action="{% url "gold_subscription" %}">
-        <input type="submit" value="{% trans "I can contribute financially" %}" />
+        <input type="submit" value="{% trans "I can contribute financially." %}" />
```

Comment:
Not sure if it's ok to include periods within a button.

Reply:
It's more like an entire sentence. That's why included that. Should I change it back the way it was?

Reply:
I don't have the arguments or the knowledge for choosing one or the other, was just a question p: But in general the PR should only solve the original issue, if you consider something else is needed is better to open a new issue, so your PR isn't blocked by discussions like this :)

Reply:
Yes. You are right. I am kind of new here. Sorry! It won't happen again! :+1:

Reply:
Nothing to worried about. Thanks for your interest on contributing.

Reply:
It's a great learning for me. Thanks for your constant help :)